static/src/stylesheets/module/crosswords/_anagram-helper.scss

```diff
@@ -0,0 +1,130 @@
+.crossword__anagram-helper-outer {
+    position: fixed;
+    box-sizing: border-box;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    padding-top: 2.5%;
+    background: #f7f7f7;
```

Comment:
You might be able to find a colour for this in our palette http://guardian.github.io/pasteup-palette/demo/